content/doc/book/security/index.adoc

```diff
@@ -9,30 +9,88 @@ ifdef::backend-html5[]
 :sectanchors:
 endif::[]
 
-Jenkins is used everywhere from workstations on corporate intranets, to high-powered servers connected to the public internet.
-To safely support this wide spread of security and threat profiles, Jenkins offers many configuration options for enabling, customizing, or disabling various security features.
-
-Many of the security options are enabled by default when passing the interactive setup wizard to ensure that Jenkins is secure.
-Others involve environment-specific setup and trade-offs and depend on specific use cases supported in individual Jenkins instances.
-
-This chapter will introduce the various security options available to Jenkins administrators and users, explaining the protections offered, and trade-offs to disabling some of them.
+Jenkins is used everywhere -- from workstations on corporate intranets
+to high-powered servers connected to the public internet.
+It is critically important to keep your Jenkins instance secure,
+both to protect your information and to avoid executing malicious code from your Jenkins instance.
```

Comment:
```suggestion
It is critically important to keep your Jenkins instance secure,both to protect your information and to avoid executing malicious code from your Jenkins instance.
```